src/pages/stacks-blockchain/sending-tokens.md

```diff
@@ -0,0 +1,207 @@
+};
+
+...
+
+const transaction = await makeSTXTokenTransfer(txOptions);
+```
+
+The generation method will need a few more pieces of information, as specified in the `txOptions` object:
+
+| Parameter   | Description                                                                                                                      | Optional |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `recipient` | The recipient Stacks address in c32check format                                                                                  | **No**   |
+| `amount`    | The amount of Stacks tokens to send denominated in microstacks                                                                   | **No**   |
+| `fee`       | The fee that the sender is willing to pay for miners to process the transaction. Denominated in microstacks                      | Yes      |
+| `nonce`     | A nonce is an integer that needs to be incremented by 1 for each sequential transaction from the same account. Nonces start at 0 | Yes      |
+| `senderKey` | A private key object                                                                                                             | Yes      |
+| `network`   | Specifies whether the transaction is meant for Stacks Mainnet or Testnet                                                         | Yes      |
+| `memo`      | A memo string to attach additional information to the transaction. This data is limited to 33 bytes                              | Yes      |
+
+### Estimating fees
+
+If not specified, the transaction builder will automatically estimate the fee. Estimated fee rate is supplied by a Stacks node so network access is required. The fee is calculated based on the estimate fee rate and the size of the transaction in bytes.
+
+Another way to estimate the fee is to use the `estimateTransfer()` function after you have constructed a transaction:
+
+```js
+estimateTransfer(transaction);
+```
+
+-> Note: By setting a fee in the transaction builder function, the automatic fee estimation step will be skipped.
+
+### Handling nonces
+
+If not specified, the transaction builder will automatically lookup the latest nonce for the sender account. Automatic nonce handling also requires network access. The nonce should be tracked locally when creating multiple sequential transactions from the same account. A Stacks node only updates the nonce once a transaction has been mined.
+
+The updated nonce for each account can be retrieved manually using the `getNonce()` function:
+
+```js
+getNonce(senderAddress);
+```
+
+## Step 4: Broadcasting transaction
+
+Next, we will broadcast the transaction to the Testnet using the `network` object we created earlier:
+
+```js
+const txId = await broadcastTransaction(transaction, testnet);
+```
+
+As soon as the `broadcastTransaction` is completed, a transaction ID is returned.
```

Comment:
Maybe worth re-iterating here that the fact you have a txId doesn't mean the transaction has completed executing.